controllers/bsl.go

```diff
@@ -134,7 +131,7 @@ func (r *DPAReconciler) ReconcileBackupStorageLocations(log logr.Logger) (bool,
 
 			// TODO: check for BSL status condition errors and respond here
 			if bslSpec.Velero != nil {
-				err := r.updateBSLFromSpec(&bsl, &dpa, *bslSpec.Velero)
+				err := r.updateBSLFromSpec(&bsl, dpa, *bslSpec.Velero)
```

Comment:
in all DPAReconciler methods, I would remove dpa as a parameter. the functions can access it through `r.dpa`, right?

Reply:
Code example of my suggestion
```diff
-func (r *DPAReconciler) updateBSLFromSpec(bsl *velerov1.BackupStorageLocation, dpa *oadpv1alpha1.DataProtectionApplication, bslSpec velerov1.BackupStorageLocationSpec) error {
+func (r *DPAReconciler) updateBSLFromSpec(bsl *velerov1.BackupStorageLocation, bslSpec velerov1.BackupStorageLocationSpec) error {
	// Set controller reference to Velero controller
-	err := controllerutil.SetControllerReference(dpa, bsl, r.Scheme)
+	err := controllerutil.SetControllerReference(r.dpa, bsl, r.Scheme)
	if err != nil {
		return err
	}
	...

	return nil
}
```

Reply:
Removed extra DPA parameters from all the functions, and made required changed.